translationCoreApps/translationNotes_Check_plugin
# Tests

#### This pull request addresses:

organized files, added coverage, added linting, stubbed tests

#### How to test this pull request:

I moved files around so make sure the code still runs.
The tests will fail because there are no real tests yet.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/translationcoreapps/translationnotes_check_plugin/80)
